dependencies_groups.gradle

```diff
@@ -50,6 +50,7 @@ ext.groups = [
                         'com.beust',
                         'com.davemorrissey.labs',
                         'com.dropbox.core',
+                        'com.soywiz.korlibs.korte',
```

Comment:
was this missing from the previous PR? :thinking:  thought the CI would catch those errors

Reply:
The CI does not call (yet) `./gradlew dokkaHtml` which support has been added by the previous PR, and this addition is to be able to call `./gradlew dokkaJavadoc` (as per the commit message)